From a5d5cf6c2d8ca2d660bbe26c36b787c2b204fa64 Mon Sep 17 00:00:00 2001
From: Ren-Pei Zeng <kamesan@chromium.org>
Date: Fri, 9 Oct 2020 16:45:21 +0800
Subject: [PATCH 5/5] camera: intel: ipu6: Fix camera count from XML file

Correct the initial value of the static camera count from XML config
files.

BUG=b:170412320
TEST=Check cameras functional on volteer.

Change-Id: Ie593021710e4a33683b535bd5cac9433364cfc4f
---
 camera/hal/intel/ipu6/src/platformdata/CameraTypes.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/camera/hal/intel/ipu6/src/platformdata/CameraTypes.h b/camera/hal/intel/ipu6/src/platformdata/CameraTypes.h
index 61119ac701..bd494a8733 100644
--- a/camera/hal/intel/ipu6/src/platformdata/CameraTypes.h
+++ b/camera/hal/intel/ipu6/src/platformdata/CameraTypes.h
@@ -171,7 +171,7 @@ struct CommonConfig {
     bool isGpuTnrEnabled;
     int cameraNumber;
 
-    CommonConfig() { xmlVersion = 1.0; isGpuTnrEnabled = false; cameraNumber = 0;}
+    CommonConfig() { xmlVersion = 1.0; isGpuTnrEnabled = false; cameraNumber = -1;}
 };
 
 struct OBSetting {
-- 
2.28.0.1011.ga647a8990f-goog

